--- Net_UserAgent_Detect-2.4.0/usr/share/pear/Net/UserAgent/Detect/APC.php~	2007-09-20 10:33:49.000000000 +0200
+++ Net_UserAgent_Detect-2.4.0/usr/share/pear/Net/UserAgent/Detect/APC.php	2007-09-20 10:34:30.000000000 +0200
@@ -17,7 +17,7 @@
 
 // $Id$
 
-require_once 'Net_UserAgent/detect.php';
+require_once 'Net/UserAgent/Detect.php';
 
 class Net_UserAgent_Detect_APC extends Net_UserAgent_Detect
 {
